components/indexer.py

```diff
@@ -51,7 +66,7 @@ class Direction(Enum):
     _chimney_direction = Direction.OFF
     _cat_flap_is_open = False
 
-    last_colour = CargoColour.NONE
+    cargo_colour = CargoColour()
```

Comment:
Do not assign mutable objects to a class variable that are supposed to be defaults. Class variables are global state. Expect things to break badly once RobotPy has tests again.